src/containers/ProductsView.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useParams } from 'react-router-dom';
+import { toaster, Spinner } from 'evergreen-ui/';
+import ProductDetails from '../components/ui/ProductDetails';
+
+const ProductsView = () => {
+  const [product, setProduct] = useState({}); // [{}, () => {}]
+  const [loaded, setLoaded] = useState(false);
+  const { id } = useParams(); // { category: 'jewelry', id: 1}
+
+  const getProductid = () => {
```

Comment:
Rename it to `getProduct`

Reply:
![image](https://user-images.githubusercontent.com/90296394/135179657-5df145c9-16f5-49fd-87b5-c9eddf5d671b.png)
Const renamed